site/content@beta/backend-apis/core-info.md

```diff
@@ -0,0 +1,155 @@
+| `Chat assigned to <agent_name> because <agent_name> hasn't replied in <minutes_number> minutes` | `routing.assigned` |
+| `Chat archived due to long inactivity` | `routing.archived` |
+| `Chat is idle due to <minutes_number> minutes of inactivity` | `routing.idle` |
+| `Chat archived because customer was banned by <agent> for N days` | `customer_banned` |
+
+#### System messages: routing.assigned
+
+##### General info
+
+| Message text | System message type |
+|--------------|---------------------|
+| `Chat assigned to <agent_name>` | `routing.assigned` |
+
+##### Use cases
+
+| Case| Routing type |
+|--------------|---------------------|
+| A new chat is started and an agent is available | automatic |
+| An agent has left the chat and another agent is available | automatic |
+| An agent got available for an unassigned chat | automatic |
+
+#### System messages: routing.unassigned
+
+##### General info
+
+| Message text | System message type |
+|--------------|---------------------|
+| `Chat is unassigned` | `routing.unassigned` |
+
+##### Use cases
+
+| Case| Routing type |
+|--------------|---------------------|
+| An agent has left the chat and there were no other agents are available | automatic |
+| No free agent slots available | automatic |
+| A chat is unassigned | manual |
+
+## Bot Agents
+
+Bot Agents are similar to their human counterparts. They can join chats and post messages, but they also have a special feature: you can attach [webhooks](../configuration-api/#webhooks) to them.
+
+<img src="images/bot-agent.jpg" alt="LiveChat Bot Agent" class="has-border"/>
+
+Bot Agents are created and managed with the use of the  [Configuration API](../configuration-api/#bot-agent). They communicate with the [Agent Chat API](../agent-chat-api/) by the [**Web API**](../agent-chat-api/#web-api) or [**websocket connection**](#rtm-api-vs-web-api). They listen to incoming webhooks (or pushes) and react to them.
+
+#### Example: reacting to keywords
+
+Bot Agents can react to specific keywords appearing in the chat. Let's say you set a _pizza_ keyword. Your Bot Agent will join the chat whenever the keyword is used and send the _"Woohoo!"_ message to all agents in the chat. Then, it'll leave the chat.
+
+### Bots vs. regular Agents
+
+Here are the major differences between Bot Agents and regular Agents:
+
+* You can't log in to a Bot Agent account.
+* You can't set password for a Bot Agent account.
+* Bot Agents don't have email addresses. Their <code>agent_id</code> is a random hash.
+* You can assign webhooks to Bot Agents as a communication channel for [pushes](../agent-chat-api/#pushes).
+
+### Technical notes
+
+* Bot Agents use the [Agent Chat API](../agent-chat-api/) to post messages to chats as Agents, so you can use them to write your own integrations. 
+
+* When logged in, a Bot Agent is connected to the Agent's SSO access token, which creates and updates the Bot. A Bot Agent is logged out when the access token is revoked.
```

Comment:
Boty się nie logują

Reply:
hm... to jest copy-paste z obecnego beta Platform Overview. niestety nie wiem zbyt duzo na temat botów wiec nic od siebie nie wymyślę. proszę w takim razie o pomoc w przerobieniu tej sekcji o botach

Reply:
this description was removed from the doc